packages/cozy-client/src/store/queries.js

```diff
@@ -64,7 +64,9 @@ const updateQueryDataFromResponse = (queryState, response, documents) => {
     const sorter = makeSorterFromDefinition(queryState.definition)
     const doctype = queryState.definition.doctype
     const allDocs = documents[doctype]
-    const docs = updatedIds.map(_id => allDocs[_id])
+    const docs = updatedIds
+      .map(_id => (allDocs ? allDocs[_id] : null))
+      .filter(Boolean)
```

Comment:
As `allDocs` impacts all the loop, would it be more readable/optimised to put the condition around the `map` instead of inside? I think it would also allow to remove the boolean filter.

Reply:
There are two things here, _id maybe not present in allDocs, in this case we'll have an undefined, and there is the allDocs check. I agree that the allDocs check can be put outside, but the boolean filter will stay.

Reply:
Done, thanks.